MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -134,13 +134,12 @@ public void Dispose()
             Module.Memory.SetPointer("NXTCMD", Module.Memory.GetVariablePointer("INPUT"));
             Module.Memory.AllocateVariable("NMODS", 0x2); //Number of Modules Installed
             Module.Memory.SetWord("NMODS", 0x1); //set this to 1 for now
-            var modulePointer = Module.Memory.AllocateVariable("MODULE", 0x4); //Pointer to Registered Module
-            var modulePointerPointer =
-                Module.Memory.AllocateVariable("MODULE-POINTER", 0x4); //Pointer to the Module Pointer
-            Module.Memory.SetArray(modulePointerPointer, modulePointer.Data);
-            Module.Memory.AllocateVariable("UACOFF", 0x4);
-            Module.Memory.AllocateVariable("EXTOFF", 0x4);
-            Module.Memory.AllocateVariable("VDAPTR", 0x4);
+            Module.Memory.AllocateVariable("MODULE", (IntPtr16.Size * 0xFF), true); //Array of Module Info Pointers
+            Module.Memory.AllocateVariable("**MODULE", IntPtr16.Size);
+            Module.Memory.SetPointer("**MODULE", Module.Memory.GetPointer("*MODULE"));
```

Comment:
`Module.Memory.GetPointer("*MODULE")`

I don't see a `*MODULE` defined?

Reply:
The overload on AllocateVariable sending in `true` will allocate a pointer variable by the same name prepending `*`